app/connectors/DepartureMovementP5Connector.scala

```diff
@@ -85,4 +85,12 @@ class DepartureMovementP5Connector @Inject() (config: FrontendAppConfig, http: H
 
     http.GET[IE060Data](serviceUrl)(implicitly, headers, ec)
   }
+
+  def getRejectionMessage(path: String)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[IE056Data] = {
+    val headers = hc.withExtraHeaders(("Accept", "application/vnd.hmrc.2.0+json"))
+
+    val serviceUrl = s"${config.commonTransitConventionTradersUrl}$path"
+
+    http.GET[IE056Data](serviceUrl)(implicitly, headers, ec)
+  }
```

Comment:
This looks the same as the method above, can we maybe bring in an ambigious type to simplify e.g.

def getMessage[A](path: String)(implicit ec: ExecutionContext, hc: HeaderCarrier, reads: Reads[A]): Future[A] = ???